client/src/route/account/AccountTabData.tsx

```diff
@@ -59,9 +60,11 @@ function ProfileTab() {
       <h2>Profile</h2>
       <img src={pfp} alt="Avatar" data-testid="profile-picture" />
       <p>
-        The username is <b>{name}</b>. See more details on{' '}
+        The username is <b>{name}</b>. You can edit your profile details on{' '}
```

Comment:
Please change the text to

`You can edit your profile details and change password on .....`